app/sidekiq/hca/ezr_submission_job.rb

```diff
@@ -75,14 +74,23 @@ def perform(encrypted_form, user_uuid)
     rescue VALIDATION_ERROR => e
       StatsD.increment("#{STATSD_KEY_PREFIX}.enrollment_system_validation_error")
 
-      PersonalInformationLog.create!(
-        data: parsed_form,
-        error_class: 'Form1010Ezr EnrollmentSystemValidationFailure'
-      )
+      PersonalInformationLog.create!(data: parsed_form, error_class: 'Form1010Ezr EnrollmentSystemValidationFailure')
 
-      Form1010Ezr::Service.new(nil).log_submission_failure(parsed_form)
+      Form1010Ezr::Service.new(nil).log_submission_failure_to_sentry(parsed_form, '1010EZR failure', 'failure')
       self.class.log_exception_to_sentry(e)
```

Comment:
I feel like having this exception call is redundant. `log_exception_to_sentry` is essentially `log_message_to_sentry` with the `:error` flag, which we use 🤷